packages/vscode-extension/src/debug/taskTerminal/prerequisiteTaskTerminal.ts

```diff
@@ -3,13 +3,24 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 import * as vscode from "vscode";
+import { checkAndInstallForTask } from "../prerequisitesHandler";
 import { BaseTaskTerminal } from "./baseTaskTerminal";
 
 export interface PrerequisiteArgs {
   prerequisites?: string[];
   ports?: number[];
 }
 
+export enum Prerequisite {
+  nodejs = "nodejs",
+  m365Account = "m365Account",
+  devCert = "devCert",
+  func = "func",
+  ngrok = "ngrok",
+  dotnet = "dotnet",
+  ports = "ports",
+}
```

Comment:
Will `PrerequisiteArgs` and `Prerequisite` be used in CLI in the future?

Reply:
`PrerequisiteArgs` may be different in CLI, `prerequisite` may be same.
What about move them into core after CLI design is ready?

Reply:
Sure.